modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.visor.ru;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import org.apache.ignite.internal.dto.IgniteDataTransferObject;
+import org.apache.ignite.internal.processors.ru.RollingUpgradeModeChangeResult;
+import org.apache.ignite.internal.util.typedef.internal.S;
+import org.apache.ignite.internal.util.typedef.internal.U;
+
+/**
+ * Represents a wrapper class for a value of {@link RollingUpgradeModeChangeResult}.
+ */
+public class VisorRollingUpgradeChangeModeResult extends IgniteDataTransferObject {
+    /** */
+    private static final long serialVersionUID = 0L;
+
+    /** Rolling upgrade status. */
+    private VisorRollingUpgradeStatus status;
+
+    /** Overall status. */
+    private RollingUpgradeModeChangeResult.Result res;
+
+    /** The reason why the operation was failed. */
+    private Exception cause;
```

Comment:
Use VisorExceptionWrapper for wrapping of exceptions.

Reply:
Fixed.